.github/workflows/update.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: update
         run: |
           nix-build repo -A autoPrUpdate
-          result/bin/auto-pr-update > body
+          result/bin/auto-pr-update repo > body
```

Comment:
Do we maybe want to add `body` to gitignore?

Reply:
Feels overkill, this should be fixed now :)